test/edu/csus/ecs/pc2/tools/PasswordGeneratorTest.java

```diff
@@ -4,8 +4,6 @@
 import java.util.List;
 
 import edu.csus.ecs.pc2.core.util.AbstractTestCase;
-import edu.csus.ecs.pc2.tools.PasswordGenerator;
-import edu.csus.ecs.pc2.tools.PasswordType2;
```

Comment:
more unrelated changes.

Reply:
Same response as to the previous comment.